nexus_constructor/about_window.py

```diff
@@ -0,0 +1,61 @@
+import os.path as osp
+
+from PySide2.QtGui import QFont
+from PySide2.QtWidgets import QLabel, QMainWindow, QVBoxLayout, QWidget
+
+
+class AboutWindow(QMainWindow):
+    """AboutWindow class"""
+
+    def __init__(self, parent=None):
+        super().__init__(parent=parent)
+        self.setWindowTitle("About")
+
+        if parent is not None:
+            parent.registerWindow(self)
+
+        background_image = osp.join(
+            osp.dirname(osp.abspath(__file__)), "../ui/icon.png"
+        )
+
+        self._central_widget = QWidget()
+        self._central_widget.setFont(QFont("monospace", pointSize=11))
+        self._central_widget.setStyleSheet(
+            f"background-image:url({background_image});"
+            "background-position:center; background-repeat:no-repeat;"
```

Comment:
Still the background image `repeats` in `y` direction

Reply:
@ebadkamil This ought to fix it:

 self._central_widget = QWidget()
        self._central_widget.setFont(QFont("monospace", pointSize=11))
        self._central_widget.setObjectName("central_widget")
        self._central_widget.setStyleSheet("QWidget#central_widget {"
            f"background-image:url({background_image});"
            "background-position:center; background-repeat:no-repeat; }"
        )

Works for me at least.
I think the issue is the children are inheriting the stylesheet, so it is not a repeat per se.

Reply:
I pushed the change, try it out.

Reply:
awesome. now it works. thanks

Reply:
Great! This PR looks fine by me now. I think the code is easily maintainable and we will probably just update the year anyway. :P Once Tobias is happy with the copyright, just ping me and I can approve it.